examples/projections/nongeo/cartesian_logarithmic.py

```diff
@@ -5,11 +5,9 @@
 **X**\ *width*\ [**l**][/*height*\ [**l**]] or
 **x**\ *x-scale*\ [**l**][/*y-scale*\ [**l**]]
 
-Give the *width* of the figure and the optional *height*.
-The lowercase version **x** is similar to **X** but expects
-an *x-scale* and an optional *y-scale*.
-Each axis with a logarithmic transformation requires **l** after
-its size argument.
+Give the *width* of the figure and the optional *height*. The lowercase version **x** is
```

Comment:
Here, perhaps `map` is more accurate than `figure`
```suggestion
Give the *width* of the map and the optional *height*. The lowercase version **x** is
```

Reply:
Yes, `plot` is better.